test/widget_test_stubs.dart

```diff
@@ -7,7 +7,7 @@ class MockWidget extends StatefulWidget {
   const MockWidget({Key? key}) : super(key: key);
 
   @override
-  _MockWidgetState createState() => _MockWidgetState();
+  State<MockWidget> createState() => _MockWidgetState();
```

Comment:
@Vinzent03 
What is this change for BTW?

Reply:
https://dart-lang.github.io/linter/lints/library_private_types_in_public_api.html

Reply:
It's a lint from the new `lints` package. I had to replace it many times in my application with the lint switch.

Reply:
Ah, nice! Wasn't aware of that!